openslides_backend/action/mixins/send_email_mixin.py

```diff
@@ -0,0 +1,155 @@
+import os
+import re
+import smtplib
+import ssl
+from contextlib import contextmanager
+from datetime import datetime
+from email.message import EmailMessage
+from email.utils import format_datetime, make_msgid
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+from lxml import html as lxml_html  # type: ignore
+from lxml.html.clean import clean_html  # type: ignore
+
+from ...shared.exceptions import ActionException
+from ...shared.interfaces.write_request import WriteRequest
+from ..util.typing import ActionResults
+
+# regular expression for validating an Email
+regex = r"[A-Z0-9._+\-ÄÖÜ]+@[A-Z0-9.\-ÄÖÜ]+\.[A-ZÄÖÜ]{2,}"
+
+SendErrs = Dict[str, Tuple[int, bytes]]
+
+
+class ConSecurity:
```

Comment:
Also, you can use an `Enum` here and instead of manually creating the `LIST` attribute, you can check if the string is present in the `__members__` map (https://docs.python.org/3/library/enum.html#iteration)

Reply:
> Also, you can use an `Enum` here and instead of manually creating the `LIST` attribute, you can check if the string is present in the `__members__` map (https://docs.python.org/3/library/enum.html#iteration)
IMO an `Enum` isn't a really good alternative, because you have to decide whether you want `name` or `value` and have to write more;-).
The list is build also from class-attributes

Reply:
You can do `class ConnectionSecurity(str, Enum)` and then use it just like a string, so you don't have to write name/value. My main argument is that you double the code/values, and although these values will probably never change (or at least not very often), code duplication is always bad.